### PR TITLE
TECH-7738 fix edit view buttons appearing in unexpected places

### PR DESCRIPTION
**Pull request by greglinch**
[TECH-7738 fix edit view buttons appearing in unexpected places](https://industrydive.atlassian.net/browse/TECH-7738)

#### Developer

- [ ] Ensure code complies with the [style guide](https://industrydive.atlassian.net/wiki/spaces/TECH/pages/639500407/Tech+Team+s+Style+Guides)
- [ ] Devise edge cases to test the bug fix or feature being merged
- [ ] Make sure the code is clean and understandable
- [ ] Ensure code is not overly inefficient
- [ ] Ensure documentation is understandable and accurate, including:
    - [ ] Code comments and doc strings
    - [ ] Technical documentation

#### Reviewer

- [ ] Ensure code complies with the [style guide](https://industrydive.atlassian.net/wiki/spaces/TECH/pages/639500407/Tech+Team+s+Style+Guides)
    - If there are few or minor issues, add inline GitHub comments
    - If there are many issues or it seems like the developer did not follow the style guide, leave a comment asking them to do so
- [ ] Devise edge cases to test the bug fix or feature being merged
- [ ] Make sure the code is clean and understandable
    - [ ] Ask for inline comments on unclear sections
- [ ] Ensure code is not overly inefficient
- [ ] Ensure documentation is understandable and accurate, including:
    - [ ] Code comments and doc strings
    - [ ] Technical documentation

### CODE REVIEW

File: templates/admin/submit_line.html [hunk 1]
```diff
@@ -4,10 +4,15 @@
...
+{% with request.get_full_path as url_path %}
+  {# Edit/View buttons should only appear on Person model "change" page #}
+  {% if 'person' in url_path and 'change' in url_path %}
+    {% if not '?edit=true' in request.get_full_path %}
+      <input type="submit" value="Edit" name="_edit" class="default">
+    {% elif '?edit=true' in request.get_full_path %}
+      <input type="submit" value="View" name="_view" class="default">
+    {% endif %}
   {% endif %}
+{% endwith %}
```
Comment (MiriamSexton) [inline templates/admin/submit_line.html]: Could we use the existing computed data?

```
{% if opts.model_name.lower == 'person' and change %}
  {# Edit/View buttons should only appear on Person model "change" page #}
    {% if not '?edit=true' in request.get_full_path %}
      <input type="submit" value="Edit" name="_edit" class="default">
    {% elif '?edit=true' in request.get_full_path %}
      <input type="submit" value="View" name="_view" class="default">
    {% endif %}
{% endif %}
```

Comment (greglinch) [reply templates/admin/submit_line.html]: Updated, thanks!
